### DISCUSSION TITLE
How to make Local Module work every time there is a change

### DISCUSSION THREAD

**Discussion by kukut** [GenieFramework/Genie.jl#631] [Q&A] [answered]
I make local module in Genie, and I call those module in Controller File like this:
```julia
include("MyModule.jl")
using .MyModule
```
and the code running well.

But if change code in file MyModule.jl, Controller not running as expected.
How to make it work properly?

**Comment by AbhimanyuAryan** [comment, marked as answer]
can you please post your code/link to the repo? It could be that you use `using YourAppModule.MyModule`

**Reply by kukut** [reply]
I change my code into `using MyGenieApp.MyModule` and it works...
Thank you...
